crypto/encrypt.go

```diff
@@ -29,40 +30,31 @@ func GenerateTwofishKey() (key TwofishKey, err error) {
 
 // EncryptBytes encrypts a []byte using the key. EncryptBytes uses GCM and
 // prepends the nonce (12 bytes) to the ciphertext.
-func (key TwofishKey) EncryptBytes(plaintext []byte) (ct Ciphertext, err error) {
-	// Create the cipher, encryptor, and nonce.
-	twofishCipher, err := twofish.NewCipher(key[:])
-	if err != nil {
-		return nil, err
-	}
-	aead, err := cipher.NewGCM(twofishCipher)
-	if err != nil {
-		return nil, err
-	}
+func (key TwofishKey) EncryptBytes(plaintext []byte) (Ciphertext, error) {
+	// Create the cipher.
+	// NOTE: NewCipher only returns an error if len(key) != 16, 24, or 32.
+	// NOTE: NewGCM only returns an error if twofishCipher.BlockSize != 16.
+	twofishCipher, _ := twofish.NewCipher(key[:])
+	aead, _ := cipher.NewGCM(twofishCipher)
+
+	// Create the nonce.
 	nonce := make([]byte, aead.NonceSize())
-	_, err = rand.Read(nonce)
+	_, err := rand.Read(nonce)
 	if err != nil {
 		return nil, err
 	}
 
 	// Encrypt the data. No authenticated data is provided, as EncryptBytes is
 	// meant for file encryption.
-	ct = aead.Seal(nonce, nonce, plaintext, nil)
-	return ct, nil
+	return aead.Seal(nonce, nonce, plaintext, nil), nil
 }
 
 // DecryptBytes decrypts the ciphertext created by EncryptBytes. The nonce is
 // expected to be the first 12 bytes of the ciphertext.
-func (key TwofishKey) DecryptBytes(ct Ciphertext) (plaintext []byte, err error) {
+func (key TwofishKey) DecryptBytes(ct Ciphertext) ([]byte, error) {
 	// Create the cipher.
-	twofishCipher, err := twofish.NewCipher(key[:])
-	if err != nil {
-		return nil, err
-	}
-	aead, err := cipher.NewGCM(twofishCipher)
-	if err != nil {
-		return nil, err
-	}
+	twofishCipher, _ := twofish.NewCipher(key[:])
+	aead, _ := cipher.NewGCM(twofishCipher)
```

Comment:
need to copy the comments from above into this function too, b/c it's a different function and people don't always read a file in order.